pkg/ebpf/c/common/filesystem.h

```diff
@@ -293,6 +331,38 @@ static __always_inline struct inode *get_inode_from_file(struct file *file)
     return READ_KERN(file->f_inode);
 }
 
+// Return which of the standard FDs point to the given file as a bit field.
+// The FDs matching bits are (1 << fd).
+static __always_inline int get_standard_fds_from_struct_file(struct file *file)
+{
+    struct task_struct *task = (struct task_struct *) bpf_get_current_task();
+    if (task == NULL) {
+        return -1;
+    }
+    struct files_struct *files = (struct files_struct *) READ_KERN(task->files);
+    if (files == NULL) {
+        return -2;
+    }
+    struct fdtable *fdt = (struct fdtable *) READ_KERN(files->fdt);
+    if (fdt == NULL) {
+        return -3;
+    }
+    struct file **fd = (struct file **) READ_KERN(fdt->fd);
+    if (fd == NULL) {
+        return -4;
+    }
```

Comment:
The READ_KERN macro was removed.
You should now use BPF_CORE_READ to do all this at once - like done here: https://github.com/aquasecurity/tracee/blob/main/pkg/ebpf/c/tracee.bpf.c#L4861